libs/pages/application/src/lib/feature/page-settings-healthchecks-feature/page-settings-healthchecks-feature.tsx

```diff
@@ -147,34 +117,54 @@ export function PageSettingsHealthchecksFeature() {
     methods.setValue('readiness_probe.current_type', defaultTypeReadiness)
     methods.setValue('liveness_probe.current_type', defaultTypeLiveness)
 
-    if (application?.healthchecks?.readiness_probe) {
-      setProbeValues('readiness_probe', application?.healthchecks?.readiness_probe)
+    if (service?.healthchecks?.readiness_probe) {
+      setProbeValues('readiness_probe', service?.healthchecks?.readiness_probe)
     }
 
-    if (application?.healthchecks?.liveness_probe) {
-      setProbeValues('liveness_probe', application?.healthchecks?.liveness_probe)
+    if (service?.healthchecks?.liveness_probe) {
+      setProbeValues('liveness_probe', service?.healthchecks?.liveness_probe)
     }
-  }, [methods, application, defaultTypeReadiness, defaultTypeLiveness])
+  }, [
+    methods,
+    service?.healthchecks?.liveness_probe,
+    service?.healthchecks?.readiness_probe,
+    defaultTypeReadiness,
+    defaultTypeLiveness,
+  ])
+
+  if (!service) return null
 
   return (
     <FormProvider {...methods}>
       <PageSettingsHealthchecks
-        ports={application?.ports}
-        jobPort={application?.port}
-        isJob={isJob(application)}
-        minRunningInstances={application?.min_running_instances}
+        ports={(service as Application | Container).ports}
+        jobPort={(service as Job).port}
+        isJob={service.serviceType === 'JOB'}
+        minRunningInstances={(service as Application | Container)?.min_running_instances}
         linkResourcesSetting={`${APPLICATION_URL(
           organizationId,
           projectId,
           environmentId,
           applicationId
         )}${APPLICATION_SETTINGS_URL}${APPLICATION_SETTINGS_RESOURCES_URL}`}
         onSubmit={onSubmit}
-        loading={loading}
+        loading={isLoadingEditService}
         environmentMode={environment?.mode}
       />
     </FormProvider>
   )
 }
 
-export default PageSettingsHealthchecksFeature
+export function PageSettingsAdvancedFeature() {
+  const { environmentId = '', applicationId = '' } = useParams()
+
+  const { data: serviceType } = useServiceType({ environmentId, serviceId: applicationId })
+
+  if (serviceType !== ('APPLICATION' || 'CONTAINER' || 'JOB')) {
```

Comment:
This doesn't work in JS, you must use `! includes`

Reply:
Unfortunately it look difficult for TypeScript if I use `includes` because it keep the types for other services

example:
<img width="979" alt="Capture d’écran 2024-01-05 à 08 57 44" src="https://github.com/Qovery/console/assets/533928/bf417d24-1a5c-4995-9e98-1845acbd26f2">


it prefer something like this: 
<img width="963" alt="Capture d’écran 2024-01-05 à 08 56 14" src="https://github.com/Qovery/console/assets/533928/a4d42473-cccf-4d67-a6c9-70cd15e854ba">